docs/sts/cosign-dep.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cosign-sts
+  namespace: cosign
+spec:
+  selector:
+    matchLabels:
+      app: cosign-sts
+  template:
+    metadata:
+      labels:
+        app: cosign-sts
+    spec:
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 65533
+      serviceAccountName: cosign-sts
+      containers:
+      - args:
+        - -c
+        - env; cosign initialize --mirror=$TUF_URL --root=$TUF_URL/root.json; while true; do sleep 86400; done
+        command:
+        - /bin/sh
+        name: cosign
+        image: quay.io/redhat-user-workloads/rhtas-tenant/rhtas-cli-1-0-beta/cosign:3df04256b64d249ad0db28521ffdeb2e4ad3a696
+        env:
+        - name: AWS_ROLE_SESSION_NAME
+          value: signer-identity-session
+        - name: AWS_REGION
+          value: us-east-1
+        - name: OPENSHIFT_APPS_SUBDOMAIN
+          value: apps.open-svc-sts.k1wl.p1.openshiftapps.com
```

Comment:
I assume we would want to remove all of the specific details related to your environment and AWS account

Reply:
done